CMakeLists.txt

```diff
@@ -9,20 +9,11 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
-# examples
-add_executable(example-get-account-info examples/getAccountInfo.cpp)
-add_executable(example-account-subscribe examples/accountSubscribe.cpp)
-add_executable(example-send-transaction examples/sendTransaction.cpp)
-add_executable(example-place-order examples/placeOrder.cpp)
+# lib
+add_subdirectory(lib)
 
-# link
-target_link_libraries(example-get-account-info ${CONAN_LIBS})
-target_link_libraries(example-account-subscribe ${CONAN_LIBS})
-target_link_libraries(example-send-transaction ${CONAN_LIBS})
-target_link_libraries(example-place-order ${CONAN_LIBS})
+# examples
+add_subdirectory(examples)
 
 # tests
-include(CTest)
-add_executable(tests tests/main.cpp)
-target_link_libraries(tests ${CONAN_LIBS})
-add_test(NAME TestMangoGroup COMMAND tests)
+add_subdirectory(tests)
```

Comment:
this breaks ctest, test paths are relative to /build

Reply:
for me ctest is not really important and ci is not using it anyways, so maybe just remove the find & add_test directives

Reply:
Fixed this for `ctest` and `.bin/test` by passing the resources directory directly to the tests.